x/clp/genesis.go

```diff
@@ -1,21 +1,53 @@
 package clp
 
 import (
+	"fmt"
 	"github.com/Sifchain/sifnode/x/clp/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
-func InitGenesis(ctx sdk.Context, keeper Keeper, bankKeeper types.BankKeeper, data types.GenesisState) (res []abci.ValidatorUpdate) {
+func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) (res []abci.ValidatorUpdate) {
```

Comment:
Do we normally name the result if we don't use the name and the name doesn't have any extra meaning?

Reply:
Thats just following the coding style set by the sdk , we can definatly remove it for now . But it might be usefull when we do have more fields in the struct